packages/webapp-libs/webapp-tenants/src/components/tenantMembersList/membershipEntry/__tests__/membershipEntry.component.spec.tsx

```diff
@@ -0,0 +1,61 @@
+import { MultitenancyTenantMembershipRoleChoices, TenantUserRole } from '@sb/webapp-api-client';
+import { currentUserFactory, fillCommonQueryWithUser } from '@sb/webapp-api-client/tests/factories';
+import { composeMockedQueryResult } from '@sb/webapp-api-client/tests/utils';
+import { screen } from '@testing-library/react';
+import { userEvent } from '@testing-library/user-event';
+
+import { membershipFactory, tenantFactory } from '../../../../tests/factories/tenant';
+import { render } from '../../../../tests/utils/rendering';
+import { MembershipEntry } from '../membershipEntry.component';
+import { updateTenantMembershipMutation } from '../membershipEntry.graphql';
+
+describe('MembershipEntry: Component', () => {
+  it('should commit update mutation', async () => {
+    const mockedId = '1';
```

Comment:
Factories are generating IDs so these vars are not necessary

Reply:
that's true but factories make them random and I need to have some kind of deterministic value in order to check if I send it in the request. That's why I think it's better to overwrite it here.

Reply:
Random ids should be also fine because you are preparing the strict request which will fail if the id param for example will different then expected but it's a minor thing